store/modules/wallet.js

```diff
@@ -0,0 +1,228 @@
+/* eslint no-shadow: "off" */
+/* eslint no-param-reassign: "off" */
+import { LIKECOIN_WALLET_CONNECTOR_CONFIG } from '@/constant/network';
+
+import {
+  WALLET_SET_ADDRESS,
+  WALLET_SET_SIGNER,
+  WALLET_SET_CONNECTOR,
+  WALLET_SET_METHOD_TYPE,
+  USER_SET_USER_INFO,
+  WALLET_SET_IS_LOGGING_IN,
+  WALLET_SET_IS_CONNECTING_WALLET,
+  AUTHCORE_SET_ACCESS_TOKEN,
+} from '../mutation-types';
+
+let likecoinWalletLib = null;
+
+const state = () => ({
+  address: '',
+  isConnectingWallet: false,
+  signer: null,
+  connector: null,
+  likerInfo: null,
+  isInited: null,
+  methodType: null,
+  loginAddress: '',
+  isLoggingIn: false,
+});
+
+const mutations = {
+  [WALLET_SET_ADDRESS](state, address) {
+    state.address = address;
+  },
+  [WALLET_SET_SIGNER](state, signer) {
+    state.signer = signer;
+  },
+  [WALLET_SET_IS_CONNECTING_WALLET](state, isConnecting) {
+    state.isConnectingWallet = isConnecting;
+  },
+  [WALLET_SET_IS_LOGGING_IN](state, isLoggingIn) {
+    state.isLoggingIn = isLoggingIn;
+  },
+  [USER_SET_USER_INFO](state, userInfo) {
+    state.likerInfo = userInfo;
+  },
+  [WALLET_SET_METHOD_TYPE](state, method) {
+    state.methodType = method;
+  },
+  [WALLET_SET_CONNECTOR](state, connector) {
+    state.connector = connector;
+  },
+};
+
+const getters = {
+  getAddress: state => state.address,
+  getSigner: state => state.signer,
+  loginAddress: state => state.loginAddress,
```

Comment:
Do we actually use `loginAddress` in like.co?

Reply:
[🔥 Remove unused code](https://github.com/likecoin/like-co/pull/1740/commits/8229839584730bb0962c16eb08fbb69511e50a87)